src/om_bootstrap/nav.cljs

```diff
@@ -89,7 +89,9 @@
    [_]
    (let [{:keys [opts children]} (om/get-props owner)
          [bs props] (t/separate Nav opts {:bs-class "nav"})
-         classes {:navbar-collapse (:collapsible? bs)}
+         classes (d/class-set {:navbar-collapse (:collapsible? bs)
```

Comment:
Gah; comments are goofy on the phone. This variable was a map, and class-set returns a string. Seems like a bug, yeah? Did you change the usage as well?